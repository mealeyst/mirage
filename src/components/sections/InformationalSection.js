import PropTypes from 'prop-types'
import styled from 'styled-components'

const InformationalSection = styled.section`
  padding: 2.0rem 1.5rem;
  background-color: ${props => props.theme.colors.lightBlue[0]};
`

InformationalSection.propTypes = {
  theme: PropTypes.shape({
    colors: PropTypes.shape({
      lightBlue: PropTypes.array
    })
  })
}

/** @component */
export default InformationalSection
