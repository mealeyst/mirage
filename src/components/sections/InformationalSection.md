```js
<InformationalSection>
  <Label>Information goes here!</Label>
</InformationalSection>
```
