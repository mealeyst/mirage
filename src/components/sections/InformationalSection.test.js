import React from 'react'
import { css } from 'styled-components'
import 'jest-styled-components'

import InformationalSection from './InformationalSection'

const { shallowWithTheme } = global

describe('(Styled component) InformationalSection', () => {
  const createInformationalSection = (props) => {
    return shallowWithTheme(<InformationalSection {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createInformationalSection())
    .toMatchSnapshot()
  })
})
