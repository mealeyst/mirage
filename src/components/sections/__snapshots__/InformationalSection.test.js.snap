// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled component) InformationalSection matching the snapshot 1`] = `
.c0 {
  padding: 2.0rem 1.5rem;
  background-color: #EEFCFF;
}

<section
  className="c0"
/>
`;
